Writerside/topics/garbage-collection.md

```diff
@@ -9,7 +9,7 @@ A field is considered stale if its **received date** is older than its (client c
 
 See [](cache-control.md) for more information about staleness.
 
-Stale fields can be removed from the cache by calling the `ApolloStore.removeStaleFields` function.
+Stale fields can be removed from the cache by calling the [`ApolloStore.removeStaleFields()`](https://apollographql.github.io/apollo-kotlin-normalized-cache-incubating/kdoc/normalized-cache-incubating/com.apollographql.cache.normalized/remove-stale-fields.html) function.
```

Comment:
Cool! PS: would be cool to have an intelliJ plugin that does this automatically. There are a bunch of places where I could use this.